### ISSUE TITLE
Cannot override default windows shortcuts anymore

### ISSUE THREAD

**Issue by palad1** [digitalcreations/MaxTo#127]
Previous versions of the app (about two years ago?) allowed me to bind win + arrows.
I can't bind this anymore, and the shortcut form does not recognize win-arrows or win-ctrl-arrows.

I had to manually edit my settings file, which you will find attached.

Thanks!
[user.zip](https://github.com/digitalcreations/MaxTo/files/106727/user.zip)

**Comment by vegardlarsen** [comment]
This is because Windows now owns that combination, and we cannot use it. MaxTo has gotten better at recognizing keyboard shortcuts that will cause trouble.

Basically, you can exit MaxTo completely, and try using Windows+Left to what happens to your active window. We cannot override this without disabling that functionality completely, unfortunately.

**Comment by mitoskalandiel** [comment]
should we not be able to decide whether we want that override or not in app-settings? my thought process here goes along the line of the fact that if we use MaxTo we really want to achieve something that windows is lacking natively and thus want to override (as a behavior) anyways

why not use something that we would start with anyways (Windows + Arrows to move windows) to "replace" rather than "add" functionality. Again, if a user decides to override this native functionality, he should be able to. For me it gets even more complicated as I rely on the NUM keys for Dexpot Virtual Desktop management.

So basically I use 3 physical screens
All three subdivided into 3 different sized chunks with MaxTo.
My most-right screen additionally features 6 virtual desktops which I switch over to for different remote sessions/protocols.

So ideally I want to be able to 

 * _[Dexpot]_ `Win+Num#` = switch to Desktop #
 * _[Dexpot]_ `Win+Alt+Num#` = move Window (under cursor) to Desktop #
 * _[MaxTo]_ `Win+ArrowDir` = Move Window to ArrowDir into the MaxTo "Frame"
 * _[MaxTo]_ `Win+End` = Maximize Window to Full Screen

But since MaxTo won't allow me to override the Win key combinations this is thwarted.

**Comment by vegardlarsen** [comment]
@mitoskalandiel MaxTo cannot override Windows keyboard shortcuts because Windows won't let us. It is that simple.

Ideally I would also like to be able to do that, but it does not seem to be possible without disable large parts of Windows' functionality.

**Comment by mitoskalandiel** [comment]
Without having this tested myself on my work machine, but would [UnregisterHotKey()](https://msdn.microsoft.com/en-us/library/ms646327(v=vs.85).aspx) not solve this issue, by simply unregistering the hotkey in question and then mapping it to your own function with RegisterHotKey()?

If this is not possible, then I give up and will have to think of an alternate mapping :)

Sorry for the inconvenience :)

**Comment by vegardlarsen** [comment]
I don't think you can UnregisterHotKey for built-in Windows hotkeys. I'd love to be proven wrong. Send me some sample code and I promise I'll have a look... :)
